website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -806,6 +808,37 @@ dataSources:
             kind: call
 ```
 
+#### Polling Filter
+
+```yaml
+blockHandlers:
+  - handler: handleBlock
+    filter:
+      kind: polling
+      every: 10
+```
+
+The defined handler will be called once for every `n` blocks, where `n` is the value provided in the `every` field. This configuration allows the subgraph to perform specific operations at regular block intervals.
+
+#### Once Filter
+
+```yaml
+blockHandlers:
+  - handler: handleOnce
+    filter:
+      kind: once
+```
+
+The defined handler with the once filter will be called only once before all other handlers run. This configuration allows the subgraph to use the handler as an initialization handler, performing specific tasks at the start of indexing.
```

Comment:
We modified the filtering system here, for this. So that block handlers can now trigger before too. `Once` trigger always before any other handler. 

Its also easy now to add a new feature, some sort of `end` filter to make normal blockHandlers trigger at the begining rather than the end

Reply:
nice!